src/main/java/fr/gravendev/multibot/moderation/events/GuildMessageReceivedListener.java

```diff
@@ -110,6 +118,21 @@ private void warnForCapitalsLettersIfNeeded(Message message) {
         }
     }
 
+    private void warnForSpamIfNeeded(Message message){
+        if(message.getContentDisplay().length() > 5) return;
+        if(spamCounter.containsKey(message.getAuthor().getId())){
+            if(spamCounter.get(message.getAuthor().getId()) == 5){
+                warn(message, "Spam");
+                spamCounter.remove(message.getAuthor().getId());
+            }else{
+                spamCounter.put(message.getAuthor().getId(), spamCounter.get(message.getAuthor().getId()) + 1);
+            }
+        }else{
+            scheduledExecutorScheduler.schedule(() -> spamCounter.remove(message.getAuthor().getId()), 3, TimeUnit.SECONDS);
+            spamCounter.put(message.getAuthor().getId(), 1);
+        }
+    }
+
```

Comment:
Please try to keep existing code style, put spaces before `{` and after `}`, and avoid duplication of `message.getAuthor().getId()`. And finally, replace the magic values (`5`, `5`, and `3`) by named constants for better readability.